lizrd/support/code_versioning.py

```diff
@@ -58,3 +91,24 @@ def reset_to_original_repo_state(
         repo.git.branch("-D", versioning_branch)
     repo.head.reset(original_branch_commit_hash, index=True)
     print("Successfully restored working tree to the original state!")
+
+
+if __name__ == "__main__":
```

Comment:
to jest w celu zversionowania kodu u siebie lokalnie? czy test którego nie wywaliłeś?